hedera-mirror-rest/service/recordFileService.js

```diff
@@ -128,6 +133,31 @@ class RecordFileService extends BaseService {
 
     return rows.map((recordFile) => new RecordFile(recordFile));
   }
+
+  async getByHashOrNumber(hash, number) {
+    let whereStatement = '';
+    const params = [];
+    if (hash) {
+      const hashWithPrefix = utils.addHexPrefix(hash);
```

Comment:
Why add prefix just to remove it again?
If the goal is just to remove the `0x` if it exists then we can do this with 
```suggestion
      const hashWithoutPrefix = hash.replace('0x', '');
```